modules/page_object_about_prefs.py

```diff
@@ -58,8 +58,13 @@ def select_option(self, option_name: str):
             if len(matching_menuitems) == 0:
                 return False
             elif len(matching_menuitems) == 1:
-                matching_menuitems[0].click()
-                self.wait.until(EC.element_to_be_selected(matching_menuitems[0]))
+                if double_click:
+                    AboutPrefs(self.driver).double_click(
```

Comment:
why is this not `self.double_click(`?

Reply:
self refers to `Dropdown` class. So I have to create an instance of `AboutPrefs` (since the `double_click` method is not static) 

Relevant logs: (when i use `self`)

```
FAILED tests/language_packs/test_language_pack_install_addons.py::test_language_pack_install_from_addons[Italiano-it-Imposta alternative\u2026] - AttributeError: 'Dropdown' object has no attribute 'double_click'
```